src/main/java/use_cases/LoadPlayer.java

```diff
@@ -0,0 +1,27 @@
+package use_cases;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class LoadPlayer {
+    /**
+     * Loads a HumanPlayer's wins and losses from "src/main/java/use_cases/players.csv" given their name.
+     * @param name The name of the HumanPlayer.
+     * @return A string array [name, wins, losses] of the HumanPlayer, or null if the name is not found.
+     * @throws IOException If an input/output error occurs when reading from the file.
+     */
+    public static String[] loadPlayerFromFile(String name) throws IOException {
+        String file = "src/main/java/use_cases/players.csv";
```

Comment:
just as an example:

```
public interface DataAccess {
    PlayerInformation load(String name, String file)  // here PlayerInformation would just be a data object of all the player stats
    void save(String name, String file)
}
```

with an example implementation:

```
public class CSVDatabase implements DataAccess {
  ...
  
  @Override
  public PlayerInformation load(String name, String file) {
    BufferedReader br = ...
    // your existing code in LoadPlayer
  }

  @Override
  public void save(String name, String file) {
    // your code in SavePlayer
  }
}
```
your existing usecases would then store a reference to a DataAccess interface and call its method, and we'd pass in a CSVDatabase class wherever we initialize them

the specifics can be decided later but i think this is a good guideline for keeping use cases separate from the databases.

Reply:
I've pushed new changes according to this suggestion. Let me know if I should make further edits.